ERCS/erc-8043.md

```diff
@@ -0,0 +1,313 @@
+---
+eip: 8043
```

Comment:
```suggestion
eip: 8056
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.

Reply:
Done